.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "src/resources/json"]
-	path = src/resources/json
+[submodule "src/main/java/resources/json"]
+	path = src/main/java/resources/json
```

Comment:
Looking at the list of changes in the PR diff, you broke the submodule, because the path `src/resources/json` is removed entirely, not moved to `src/main/java/resources/json`

Reply:
Note that this probably the reason why Travis is broken.

Reply:
I re-added the submodule correctly, but didnt check tests locally (yet). Thx for pointing it out!

Reply:
We still need a way to have a useful test output on Travis though